vulcanai2/tests/models/test_cnn.py

```diff
@@ -1,66 +1,44 @@
 """Test all ConvNet capabilities."""
-import pytest
 import numpy as np
+import pytest
+
 import torch
+import torch.nn as nn
+from torch.utils.data import DataLoader, TensorDataset
+
+from vulcanai2.models import BaseNetwork
 from vulcanai2.models.cnn import ConvNet
-from torch.utils.data import TensorDataset, DataLoader
+from vulcanai2.models.utils import master_device_setter
 
 
 class TestConvNet:
     """Define ConvNet test class."""
+    
+    def test_init_ConvNet(self, conv1D_net):
+        """Initialization Test of ConvNet """
+        assert isinstance(conv1D_net, BaseNetwork)
+        assert isinstance(conv1D_net, nn.Module)
+        assert hasattr(conv1D_net, 'input_networks')
+        assert hasattr(conv1D_net, 'network')
+        assert hasattr(conv1D_net, 'in_dim')
+        assert hasattr(conv1D_net, 'device')
+    
+    def test_forward_ConvNet(self, conv1D_net):
+        """Test Forward of ConvNet"""
+        out = conv1D_net(torch.ones([10, *conv1D_net.in_dim]))
+        assert out.shape == (10, *conv1D_net.out_dim)
 
-    @pytest.fixture
-    def cnn_noclass(self):
-        """Create ConvNet with no prediction layer."""
-        return ConvNet(
-            name='Test_ConvNet_noclass',
-            in_dim=(1, 28, 28),
-            config={
-                'conv_units': [
-                    {
-                        "in_channels": 1,
-                        "out_channels": 16,
-                        "kernel_size": (5, 5),
-                        "pool_size": 2,
-                        "stride": 2
-                    },
-                    {
-                        "in_channels": 16,
-                        "out_channels": 1,
-                        "kernel_size": (5, 5),
-                        "stride": 2,
-                        "padding": 2
-                    }]
-            }
-        )
+    def test_forward_multi_input_ConvNet(self, multi_input_cnn,
```

Comment:
you don't need to include ConvNet in the title since it already exists within the class `TestConvNet`